Sources/Afluent/SequenceOperators/DelaySequence.swift

```diff
@@ -8,6 +8,7 @@
 import Foundation
 
 extension AsyncSequences {
+    @available(iOS 16, *)
```

Comment:
Can you please add all other OS requirements? macOS, tvOS, watchOS, etc.

Reply:
Yep! On it.

Reply:
Pushed